src/app_service/service/contacts/service.nim

```diff
@@ -68,11 +70,16 @@ QtObject:
     self.contactsStatus.clear
     self.QObject.delete
 
-  proc newService*(events: EventEmitter, threadpool: ThreadPool): Service =
+  proc newService*(
+      events: EventEmitter,
+      threadpool: ThreadPool,
+      settingsService: settings_service.Service
+    ): Service =
```

Comment:
nit: indent, make it the way you prefer but pls differentiate the code of the function with the declaration